src/wp-includes/class-wp-rewrite.php

```diff
@@ -52,39 +52,39 @@ class WP_Rewrite {
 	 * Permalink structure for author archives.
 	 *
 	 * @since 1.5.0
-	 * @var string
+	 * @var ?string
 	 */
 	public $author_structure;
 
 	/**
 	 * Permalink structure for date archives.
 	 *
 	 * @since 1.5.0
-	 * @var string
+	 * @var ?string
 	 */
 	public $date_structure;
 
 	/**
 	 * Permalink structure for pages.
 	 *
 	 * @since 1.5.0
-	 * @var string
+	 * @var ?string
 	 */
 	public $page_structure;
 
 	/**
 	 * Base of the search permalink structure (example.com/$search_base/query).
 	 *
 	 * @since 1.5.0
-	 * @var string
+	 * @var ?string
```

Comment:
I don't think this change is correct. I don't see this ever getting unset.

```suggestion
	 * @var string
```

Reply:
Reverted in 8710a699b7756a8d48e95691f3939e570330325c